src/utils.js

```diff
@@ -46,6 +46,8 @@ const stringReplace = unapply(String.prototype.replace);
 const stringIndexOf = unapply(String.prototype.indexOf);
 const stringTrim = unapply(String.prototype.trim);
 
+const objectHasOwnProperty = unapply(Object.prototype.hasOwnProperty);
```

Comment:
JavaScript does not protect the property name `hasOwnProperty`, so objects with properties with this name may return incorrect results.

This is a more stable way to use it.

### Related
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#using_hasownproperty_as_a_property_name)
[eslint no-prototype-builtins](https://eslint.org/docs/latest/rules/no-prototype-builtins)